packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -23,6 +23,28 @@ location / {
     include /opt/mesosphere/etc/adminrouter-ui-security.conf;
 }
 
+location /assets {
+    # The UI asset filenames are hashed by content. Hence, they can be cached
+    # by the client indefinitely. Currently, we use a more conservative value.
+    expires 10h;
```

Comment:
Do we really want to mix this new caching concept into the current PR?

Reply:
Since we're aiming for faster responsiveness of the UI, setting good cache headers is important. There isn't anything faster than finding the file in the cache already and knowing that it is fresh.

Having said that, we should probably either remove the caching to keep it in a separate PR/discussion, or set the right values: e.g. 1 minute for `index.html` so it updates when the UI is updated, and up to a year for hashed files.